### PR TITLE
Fixed chips and small screen layouts

### PR DESCRIPTION
**Pull request by kungpaogao**
### Summary 

This pull request addresses issues #26 and #39.  

Fixes for chips:

- chip text color stays black when selected
- chip text is centered (regardless of length of label)

Fixes for smaller screens: 

- chip group for historical data is now in a `HorizontalScrollView`, so all the days can still be accessed even on a small screen
- new constraints for the facility hours makes sure that text doesn't overlap

### Test Plan 

App behaves like it should before we bumped some versions. This time, we actually tried using the app on a smaller screen (Nexus 5 emulator).

### CODE REVIEW

File: Density/app/src/main/res/values/styles.xml [hunk 1]
```diff
@@ -36,10 +36,6 @@
         <item name="queryBackground">@android:color/transparent</item>
     </style>
 
-    <style name="ChipTextStyle">
-        <item name="android:textSize">13sp</item>
-    </style>
-
```
Comment (SamChou19815) [inline Density/app/src/main/res/values/styles.xml]: Why is this removed and the other xml uses the hardcoded value `13sp`?

Comment (kungpaogao) [reply Density/app/src/main/res/values/styles.xml]: I think I might add it back as a value instead of a style because having a whole style just to set text size seems unnecessary.
